eng/tox/allowed_pylint_failures.py

```diff
@@ -60,5 +60,6 @@
     "azure-messaging-nspkg",
     "azure-agrifood-farming",
     "azure-ai-language-questionanswering",
-    "azure-ai-language-conversations"
+    "azure-ai-language-conversations",
+    "azure-eventhub"
```

Comment:
Could we open an issue to re-enable any analysis/test passes we've opted out of?

Reply:
issue created: https://github.com/Azure/azure-sdk-for-python/issues/22455